lib/Controller/PageController.php

```diff
@@ -221,10 +221,16 @@ public function compose(string $uri): RedirectResponse {
 			function (&$value, $key) {
 				$value = "$key=" . urlencode($value);
 			});
-
-		$hashParams = '#mailto?' . implode('&', $params);
```

Comment:
you still need the `$params`. You can try passing them as second argument of `linkToRoute`  because we might add them to the URL then. If that doesn't work append this to the `$baseUrl` like before but without the `#mailto?` part :)